sys/linux/bpf.txt

```diff
@@ -400,19 +408,25 @@ bpf_obj_get {
 	file_flags	flags[bpf_open_flags, int32]
 }
 
+bpf_attach_targets [
+	zero	const[0, int32]
+	cgroup	fd_cgroup
+	map	fd_bpf_map
+] [size[32]]
+
 bpf_attach_arg {
-	target_fd	fd_cgroup
+	target_fd	bpf_attach_targets
```

Comment:
I tried using `bpf_attach_targets[opt]` instead to avoid the `zero` member above, but that didn't work.